common/config/rush/variants/stable/pnpmfile.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/**
```

Comment:
Err... why is this file needed in our case? Please remove or document.

`readPackage` doesn't have any overrides at all...

Reply:
That was required by rush variant, according to their instruction:
https://rushjs.io/pages/advanced/installation_variants/#:~:text=shrinkwrap.yaml%2C%20npm%2Dshrinkwrap,as%20your%20package%20manager